kubernetes/istio-install/istio-mixer.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
```

Comment:
It would seem totally reasonable for source.service to have the value nil or "none" when the caller is not a service. But it still lets one identify calls that are from a particular service. Maybe source.name is intended for this purpose? I'm not sure I'm understanding what value source.name is intended to have when the call is from a service. 

Say I want to rate-limit calls to the `ratings` service from the `reviews` service `test` version. I would have imagined something like `selector: target.service == "ratings.*" && source.service == "reviews.*" && "test" in source.labels`. Is something like this not what you guys are imagining?